### PR TITLE
initials.addTo('test.test@test.org <test.test@test.org>')

### LINKED ISSUE #7: test.test@test.org <test.test@test.org>
should become `test.test@test.org (tt) <test.test@test.org>`, but becomes ` test test test org (test.test@test.org <test.test@test.org>) <test.test@test.org> `

### PR DESCRIPTION
**Pull request by gr2m**
I've added the test. Needs implementation

closes #7

### CODE REVIEW

File: initials.js [hunk 1]
```diff
@@ -239,6 +239,19 @@
     if (email) {
       // if yes, remove it from name
       name = name.replace(email, '');
+
+      // if the email and the name are the same, initials can not be rendered
+      // the initials method uses email for rendering just when the name is false
+      // see https://github.com/gr2m/initials/issues/7 for more details
+      if (name.trim() === '<' + email + '>') {
+        // set the name to undefined
+        name = name.replace(name, '');
```
Comment (gr2m) [inline initials.js]: why not simply 

```ls
name = ''
```

?

**Comment by gr2m** [comment]
@ola can you please merge the two `test: ...` commits, and reword the `fix: ...` to something like `fix: ignore name when name === email #7`

File: initials.js (hunk 1 above)
Comment (zoepage) [inline initials.js]: because I'm a bit paranoid, I guess :D
will change :)

**Comment by gr2m** [comment]
great :clap: :+1:

**Pull request merged**